edx_proctoring/tests/test_views.py

```diff
@@ -85,8 +85,11 @@ def test_no_anonymous_access(self):
                         try:
                             response = self.client.get(reverse(name, args=["0/0/0"]))
                         except NoReverseMatch:
-                            # some require 2 args.
-                            response = self.client.get(reverse(name, args=["0/0/0", 0]))
+                            try:
```

Comment:
Can you explain why we need to make this change? Very weird code and difficult to read. Add some comments?

Reply:
This was the one test that was failing. I think it failed because the arguments used for this test did not match the regex on the new endpoint I added, so I added another case to fit the regex for an exam id and a user id. I'll add a comment to explain that